library/ssl_srv.c

```diff
@@ -3786,16 +4438,16 @@ static int ssl_parse_certificate_verify( mbedtls_ssl_context *ssl )
     defined(MBEDTLS_SSL_PROTO_TLS1_1)
     if( ssl->minor_ver != MBEDTLS_SSL_MINOR_VERSION_3 )
     {
-        md_alg = MBEDTLS_MD_NONE;
+        md_alg  = MBEDTLS_MD_NONE;
```

Comment:
Double space

Reply:
That's intentional to align this assignment with the next one.